internal/oem/oem.go

```diff
@@ -76,6 +76,7 @@ func (c Config) DefaultUserConfig() types.Config {
 }
 
 var configs = registry.Create("oem configs")
+var yes = &[]bool{true}[0]
```

Comment:
Sure, if it's worth the additional import.  I checked and that function has zero callers.